coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,19 @@
-class InvalidFilterException(Exception):
-    pass
+class InvalidFilterException(BaseException):
+
+    def __init__(self, filter_name, available_filters):
+        self.filter = filter_name
+        self.available_filters = list(available_filters.split(', '))
```

Comment:
I think you should already accept a list, instead of a comma-separated string^^ And below you can construct the string with `', '.join(available_filters) `